### DISCUSSION TITLE
can HTMX be used with PHP?

### DISCUSSION THREAD

**Discussion by emicheldev** [bigskysoftware/htmx#706] [Q&A] [answered]
Hello to all,
Please can HTMX be used with PHP?
Because all the tutorials I see are with Jango.
Thanks

**Comment by 1cg** [comment]
Hi There,

Yep, htmx is backend agnotstic: if your backend can produce HTML (and php can) then you can use htmx with it.

I've posted your question to the discord chat as well to see if someone with direct php/htmx experience can chime in.

Cheers,
Carson

**Comment by craigrodway** [comment, marked as answer]
As a PHP developer: yes it can. 

As @1cg says - the backend needs to respond with HTML, which PHP is of course excellent at doing.

**Comment by jreviews** [comment]
You can also take a look at:

- https://github.com/clickfwd/yoyo
- https://putyourlightson.com/plugins/sprig

**Reply by emicheldev** [reply]
@jreviews wonderfull 
Cool Thanks

**Comment by iocouto** [comment]
We program extensively with **[Kirby CMS](https://getkirby.com)**, a modern, PHP-based CMS which has many amazing features, among which is the fact that you can return any of your site's content in different formats, called '[content representations](https://getkirby.com/docs/guide/templates/content-representations)' - eg., so you can return that blog article as JSON, as XML, or as an HTML snippet for displaying article 'cards' in a 'related articles' section on the page. It makes implementing things like lists, pagination and searching on the frontend using htmx an absolute joy.
